javaparser-core/src/main/javacc-support/com/github/javaparser/ModifierHolder.java

```diff
@@ -12,11 +12,11 @@
  * Helper class for {@link GeneratedJavaParser}
  */
 class ModifierHolder {
```

Comment:
Is this class still relevant?

Reply:
It is produced when parsing modifiers and annotations. It holds both. This is where we need to have the discussion whether we want annotations and modifiers stored separately, or if we want to make them the same thing... Maybe it's better to keep them apart for now, it would make the API very "unspecific."